### DISCUSSION TITLE
Is Cash on Delivery implemented?

### DISCUSSION THREAD

**Discussion by gleniat** [saleor/saleor-checkout#135] [General]
Hi,

is Cash on Delivery payment option already implemented? If not yet, is it on the roadmap?

**Comment by timuric** [comment]
@gleniat yes, you can do cash on delivery, please check https://docs.saleor.io/docs/3.x/developer/checkout#finalizing-checkout-with-saleor-app

**Reply by dxrvip** [reply]
How does it work

**Reply by dxrvip** [reply]
Can you provide a case

**Reply by timuric** [reply]
you can add payment gateway that would simply create orders without creating transactions, when you will receive cash it is just a matter of calling`transacationCreate` in Saleor. Currently, our order confirmation page is showing orders as unpaid, so you would need to modify that. Perhaps we will add in future a tutorial or create a feature.

**Reply by dxrvip** [reply]
ok. thank you.
